### PR TITLE
Refactor Number and String tests to use chain.assertOK

### PR DESCRIPTION
**Pull request by tsukasaI**
Issue: #345 

Description:

Refactor tests
- string_test
- number_test

### CODE REVIEW

File: number_test.go [hunk 1]
```diff
@@ -315,25 +303,13 @@ func TestNumber_InDelta(t *testing.T) {
...
-					InDelta(tc.value, tc.delta).
-					chain.assertNotFailed(t)
-			} else {
-				NewNumber(reporter, tc.number).
-					InDelta(tc.value, tc.delta).
-					chain.assertFailed(t)
-			}
+			NewNumber(reporter, tc.number).
+				InDelta(tc.value, tc.delta).
+				chain.assertOK(t, tc.isInDelta)
```
Comment (gavv) [inline number_test.go:308]: nit: could you please reformat everything to a more compact 2-line format like this:

```go
			NewNumber(reporter, tc.number).InDelta(tc.value, tc.delta).
				chain.assertOK(t, tc.isInDelta)
```

**Review by gavv — COMMENTED** [review]
Thanks for PR! One small comment

File: number_test.go (hunk 1 above)
Comment (tsukasaI) [reply number_test.go:310]: thsnk you for athe comment.
reformated!

Comment (gavv) [reply number_test.go]: Sorry, I actually meant to switch from 3-line to 2-line format, not vise versa :)

I've pushed a commit: https://github.com/gavv/httpexpect/pull/347/commits/88fd692a0d3dc9dedb12d00157e65da147ff1ab6

**Comment by gavv** [comment]
Thank you!

**Pull request merged**